e2e-tests/test-display-themes.spec.js

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '../playwright';
+test('Verify Display Themes', async ({ page }) => {
+  await page.getByLabel('Open Preferences').click();
+  await page.getByRole('tab', { name: 'Display' }).click();
+  await page.getByLabel('Light').check();
+  await expect(page.getByText("Light")).toHaveCSS("color", "rgb(52, 52, 52)");
+  await page.getByLabel('Dark').check();
+  await expect(page.getByText("Dark")).toHaveCSS("color", "rgb(204, 204, 204)");
+  await page.getByLabel('System').check();
+  for (const systemTheme of ['light', 'dark']) {
+    await page.emulateMedia({ colorScheme: systemTheme });
+    if (systemTheme === 'light') {
+      await expect(page.getByText("Light")).toHaveCSS("color", "rgb(52, 52, 52)");
```

Comment:
Instead of just checking the color of a text, I believe a better check would be to ensure that the background and the color of the body tag is changed as expected.